e2e/tests-dfx/update_settings.bash

```diff
@@ -25,40 +25,40 @@ teardown() {
     ALICE_WALLET=$(dfx --identity alice identity get-wallet)
     BOB_WALLET=$(dfx --identity bob identity get-wallet)
 
-    dfx canister create hello
+    dfx canister --wallet "${ALICE_WALLET}" create hello
     dfx build hello
-    dfx canister install hello
```

Comment:
Shouldn't this work without passing `--wallet` (therefore using the default identity)?

Can we leave this test along (removing `--no-wallet` as needed), and instead add another that adapts this test for the case when `--wallet` is passed to canister create?